about_us.css

```
@@ -30,7 +30,7 @@
     border-radius: 50px;
 }
 
-.about__us__content__lettering {
+.about__us__content-lettering {

```

вот так будет правильнее, т.к. в БЭМ не может быть элемента у элемента

```suggestion
.about__us-content-lettering {

```

---

исправил - залил исправленное